site/content/docs/5.1/components/tags.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+title: Tags
+description: Documentation and examples for tag component.
+group: components
+toc: true
+---
+
+## Examples
+
+A tag is basically a rounded container that can contain some elements. It can be clickable and have a close button.
+
+{{< callout warning >}}
+Each tag version shouldn't be used aside another version of a tag.
```

Comment:
Why?

Reply:
It shouldn't be because they have different behaviors, I tried something to explain it a bit better. FI, it was a decision of both design and accessibility, since they have the same rendering and not the same behavior.